transform-plugins/docs/ValueMapper-transform.md

```diff
@@ -0,0 +1,82 @@
+# ValueMapper Transform
+
+
+Description
+-----------
+Value Mapper is a transform plugin that maps string values of a field in the input record to mapping value using mapping dataset.
```

Comment:
Please make the lines shorter (max 120, though 100 is better) as then GitHub doesn't wrap them when we review...
```
Value Mapper is a transform plugin that maps string values of a field in the input record to 
mapping value using a mapping dataset.